README.md

```diff
@@ -5,7 +5,7 @@ It allows you to add functionality to your applications to
 scan plain text and files in order to detect different categories of information. You can leverage any of
 the detectors in Nightfall's pre-built library, or you may programmatically define your own custom detectors. 
 
-Additionally, this library provides convenience features such as encapsulating the steps to chunk and upload files.
+Additionally, this library provides convenient features including the steps to chunk and upload files.
```

Comment:
```suggestion
Additionally, this library provides convenient features including a streamlined function to manage the multi-stage file upload process.```
?

Reply:
i guess what we want to say here is that if you use our library, you get to write one line of code `scanFile(myFile)` instead of having to write a large block where you have to make several serial requests in order to upload, validate, and then scan the file.